traffic_ops/traffic_ops_golang/crconfig/deliveryservice.go

```diff
@@ -491,12 +492,14 @@ order by dr.set_number asc
 		if err := rows.Scan(&pattern, &ttype, &dstype, &setnum, &dsname); err != nil {
 			return nil, nil, errors.New("scanning deliveryservice regexes: " + err.Error())
 		}
-
+		if _, ok := dsNameOrderMap[dsname]; !ok {
```

Comment:
do you think you can add a comment here to describe what you're doing (removing gaps)